### ISSUE TITLE
OOTB .net core SDK 2.1.200 console- and class library projects compile with errors

### ISSUE THREAD

**Issue by livarcocc** [NuGet/Home#6975]
_From @stigperez on May 27, 2018 7:55_

## Steps to reproduce

1. Download and install "dotnet-sdk-2.1.200-win-gs-x64.exe"
2. Open a command prompt, create a project directory under c: and navigate to the newly created directory
3. Execute "dotnet new classlib"
4. Execute "dotnet build"

## Expected  behavior
The project compiles without errors

## Actual behavior
The command prompt displays 24 compile errors in the two categories "The type or namespace name '[namespace]' could not be found" and "Predefined type '[type]' is not defined or imported". The referenced [namespace] and [type] are all referring to "System" or derived types of "System"

## Environment data
`dotnet --info` output:
.NET Command Line Tools (2.1.200)

Product Information:
 Version:            2.1.200
 Commit SHA-1 hash:  2edba8d7f1

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.17134
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.200\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.7
  Build    : 2d61d0b043915bc948ebf98836fefe9ba942be11

All my findings have been made on a newly restored laptop running Windows 10 Pro. I was able to generate a working class library on another laptop using the same procedure, and after comparing the two project folders there are several differences, mostly in obj\project.assets.json. I have attached the [two project folders](https://github.com/dotnet/cli/files/2042347/compile-test.zip) so you should be able to view the differences yourselves. Most noticeably I will simply mention the "version" key under "frameworks" seems to be invalid syntax and also a lot of dll-references are missing from the same file.

It may still be an issue on my laptop but any help would be appreciated :-)

_Copied from original issue: dotnet/cli#9353_

**Comment by livarcocc** [comment]
Moving to nuget since this seems like a restore issue.

**Comment by nkolev92** [comment]
One difference among the projects here is the package version for the NetStandard library.

In the working case it's 2.0.0
In the non-working case it's 2.0.1. 

The problem arises from the targets files not being selected by NuGet. 

```
        "build": {
          "build/netstandard2.0/NETStandard.Library.targets": {}
        }
```
is missing in the non working case. 

Now I tried an exact repro like recommended on a VM and did not see any problems there. 
The Fallback folder contained the exact and correctly extracted version of the .NETStandard package.

Some options are that the folder somehow got corrupted or failed during installation. 
We had seen a problem where the EnumerateFiles API returned the wrong thing, but that was never an issue on Windows rather on RHEL. 

@stigperez Can you please check whether this file
`C:\Program Files\dotnet\sdk\NuGetFallbackFolder\netstandard.library\2.0.1\build\netstandard2.0\NETStandard.Library.targets` exists?

Additionally, does deleting the assets file and restoring again fix the problem?

**Comment by stigperez** [comment]
Hi nkolev92 sorry for the delay, my laptop is being restored at the vendor so I won't be able to try this out for some time. I'll let you know when I get a chance to repeat the setup.

**Comment by stigperez** [comment]
Hi @nkolev92  I tried to reproduce the issue but "unfortunately" I now get a successful compilation. I should say the machine was completely wiped and although I installed the same apps and core version I no longer get the issue. I had a fair amount of issue with the laptop prior to the restore so something must have gone wrong when installing .net core.

Thank you @livarcocc and @nkolev92 for taking the time to look into this - you are free to close it :-)
